src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/02-linux-installation.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Linux Installation Guide
+description: Use this template to create a proposal for a new SplashKit tutorial.
+sidebar:
+  order: 2
+  label: 2. Linux Installation
+  attrs:
+    class: linux
+---
+
+This is a guide on how to install Splashkit on Ubuntu, along with all the necessary dependencies and software required to run Splashkit. This guide is intended to get you set up and ready to contribute to the SplashKit team if you're a Linux user. This installation guide was done on Ubuntu 24.04 LTS.
+
+## Installing Necessary Packages
+
+### Update your system
+
+To start, open up a terminal with the keys `ctrl + Alt + T`, then type in the following commands:
+
+```bash
+sudo apt update && sudo apt upgrade -y
+```
+
+![Terminal](/splashkit/linux-installation-fig2.png)
+
+### Install curl and git
+
+Curl and git can be installed with the following command:
+
+```bash
+sudo apt-get install curl git
+```
+
+![Terminal](/splashkit/linux-installation-fig3.png)
+
+## Install SplashKit
+
+### Collect the Installation scripts for SplashKit
+
+Run the following command to download the SplashKit installation scripts:
+
+```bash
+bash <(curl -s https://raw.githubusercontent.com/splashkit/skm/master/install-scripts/skm-install.sh)
+```
+
+![Terminal](/splashkit/linux-installation-fig4.png)
+
+Then you must **RESTART** you terminal.
+
+### skm Command
+
+To install SplashKit, you can use the `skm` command. Run the following command to install SplashKit:
+
+```bash
+skm
+```
+
+![Terminal](/splashkit/linux-installation-fig5.png)
+
+### skm Linux Install
+
+Then to install skm on SplashKit you will run the following command:
+
+```bash
+skm linux install
+```
+
+![Terminal](/splashkit/linux-installation-fig6.png)
+
+This command may or may not require you to enter your password to install additional dependencies for SplashKit.
+
+![Terminal](/splashkit/linux-installation-fig7.png)
+
+### skm Global Install
+
+To install SplashKit globally, run the following command:
+
+```bash
+skm global install
+```
+
+![Terminal](/splashkit/linux-installation-fig8.png)
+
+## Installing Visual Studio Code
+
+Next you will need to install visual studio code, if you have this already installed, then move to installing the necessary extensions step.
```

Comment:
Follow the same wording as the Windows guide. The windows guide assumes that the user does not have VS Code. 

Whatever wording you use there, I would use here also. Keep language procedural:
Do X
Do Y
Do Z

No need for "next do..."

Reply:
I'm not a fan of this kind of language, "next do" comes across more natural for a walk through tutorial guide.

Reply:
The language you decide on is ultimately your prerogative, however the point remains with respect to keeping the language in line with the Windows guide. 

Recommend either changing the Windows guide to say "if you have VS Code already installed, skip..." or update this to have it so the user will follow through top to bottom.